xcell/mappers/mapper_P15CIB.py

```diff
@@ -36,12 +37,14 @@ def __init__(self, config):
     def _get_hm_maps(self):
         if self.hm1_map is None:
             hm1_map = hp.read_map(self.file_hm1)
+            hm1_map = rotate_map(hm1_map, self.rot)
             self.hm1_map = [hp.ud_grade(hm1_map,
                             nside_out=self.nside)]
             self.hm1_map[0][self.hm1_map[0] == hp.UNSEEN] = 0.0
```

Comment:
If there are `UNSEEN` pixels, these should be set to 0 before rotating (otherwise they will propagate everywhere with the SHTs)